backend/dataall/modules/maintenance/api/queries.py

```diff
@@ -1,9 +1,15 @@
 """The module defines GraphQL queries for the Maintenance Activity>"""
 
 from dataall.base.api import gql
+from dataall.base.utils.enum_utils import generate_enum_query
+from dataall.modules.maintenance.api.enums import MaintenanceModes, MaintenanceStatus
 from dataall.modules.maintenance.api.resolvers import get_maintenance_window_status
 
 
 getMaintenanceWindowStatus = gql.QueryField(
     name='getMaintenanceWindowStatus', type=gql.Ref('Maintenance'), resolver=get_maintenance_window_status
 )
+
+queryMaintenanceModes = generate_enum_query(MaintenanceModes, 'Maintenance')
+
+queryMaintenanceStatus = generate_enum_query(MaintenanceStatus, 'Maintenance')
```

Comment:
Instead of generating manually for each enum should we do it automatically [here](https://code.amazon.com/packages/dataall/blobs/1cec5fe5f3196634a709a11cfc9fe7dc87051d7a/--/backend/dataall/base/api/__init__.py#L32)?

Reply:
>>You lack permission to view this package. 

I didn't want automatically expose all enums to frontend. And if we need them , we have to explicitly write it.